tests/test_cells.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import pytest
+import vtk  # noqa: F401
```

Comment:
Now that we're not using `eval` this should not be an unused import, right?
```suggestion
import vtk
```

Reply:
```suggestion
```